packages/assets-controllers/src/TokensController.ts

```diff
@@ -264,13 +264,15 @@ export class TokensController extends BaseController<
    * @param address - Hex address of the token contract.
    * @param symbol - Symbol of the token.
    * @param decimals - Number of decimals the token uses.
+   * @param name - Name of the token.
    * @param image - Image of the token.
    * @returns Current token list.
    */
   async addToken(
     address: string,
     symbol: string,
     decimals: number,
+    name?: string,
     image?: string,
```

Comment:
Instead of adding another argument smack-dab in the middle of the argument list, for backward compatibility, what do you think about adding it at the end? That way it can be left off and nothing bad will happen:

``` suggestion
    image?: string,
    name?: string,
```

Alternatively, if we don't mind introducing a backward incompatible change, what do you feel about putting `image` and `name` into an object? Generally speaking options are easier to work with inside of an object than as positional arguments. Otherwise, someone who wants to _just_ pass `image` but not `name` would have to pass `undefined`, which feels kind of ugly.

In other words perhaps we want to say:

``` suggestion
    { name, image }: { name?: string: image?: string } = {}
```

Reply:
Amazing suggestion! Did the refactor!